new/LIT7239Miscarriage.xml

```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7239Miscarriage" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Prayer against vaginal haemorrhage and miscarriage</title>
+                <title xml:lang="gez" xml:id="t2">ዓምደ፡ ብርሃን፡ መስረተ፡ ጽድቅ፡</title>
```

Comment:
typo (in the catalogue) for መሰረተ? Maybe to be normalized here and normalized or marked up with sic below